docs/run/sessions.mdx

```diff
@@ -132,17 +135,52 @@ words, a deactivated session does not block the creation of other
 sessions. Jobs from this new session would then take priority until it
 is deactivated or closed, at which point normal job selection resumes.
 
+To find the interactive timeout value for a session, follow the instructions in [Determine session details](#session-details).
+
+<span id="session-details"></span>
+## Determine session details
+
+You can find details about a session by using the `session.details()` method, from the [Quantum Platform Jobs page](https://quantum-computing.ibm.com/jobs), or from the IBM Cloud Jobs page, which you access from your [Instances page](https://cloud.ibm.com/quantum/instances). From the session details you can determine the [maximum](#max-ttl) and [interactive](#ttl) time to live (TTL) values, its status, whether it's currently accepting jobs, and more. 
+
+Example:
+
+``` python
+from qiskit_ibm_runtime import QiskitRuntimeService
+
+service = QiskitRuntimeService()
+
+with Session(service=service, backend="ibmq_qasm_simulator") as session:
+  session.details()
+  ...
+```
+
+
 <span id="ends"></span>
 ## What happens when a session ends 
 
-A session ends by reaching its maximum timeout value or when it is
-manually closed by the user. Do not close a session until all jobs
-**complete**. See [Close a session](run-sessions#close) for details. After a session is closed, the following occurs:
-
--   Any jobs that are already running continue to run.
--   Any queued jobs remaining in the session are put into a failed state.
--   No further jobs can be submitted to the session.
--   The session cannot be reopened.
+A session ends by reaching its maximum timeout value or by exiting the context manager. See [Close a session](run-sessions#close) for details. What happens when a session closes depends on how the session ended.
+
+If the maximum timeout value was reached:
+    -   Any jobs that are already running are canceled.
```

Comment:
That was my understanding from this statement "The session won't accept more jobs but it will continue to run any queued jobs until they are done or the max time expires (and then the same closed rules apply as above)." From this issue: https://github.ibm.com/IBM-Q-Software/requirements-and-priorities/issues/207#issuecomment-63469033 - but I would also appreciate clarification.

Reply:
Matt confirmed running jobs don't get cancelled.

Reply:
So even if you go over the session max time, all queued jobs will finish, @jyu00 @mriedem ?

Reply:
Queued jobs will be cancelled, but jobs already running won't.